client/src/pages/WatingRoom.tsx

```diff
@@ -0,0 +1,29 @@
+import { Typography } from '@mui/material';
```

Comment:
You have extra margins/paddings that break your mobile styles, this will make it look weak in your presentation.
Fix it if you like it to look better.
---
![Screenshot from 2023-06-23 11-24-06](https://github.com/GSG-G13/MenuMingle/assets/77394697/cdbd8425-33f8-4d2f-9444-cccb5a6fd977)